### ISSUE TITLE
Remove links for downloading metadata on MyVault

### ISSUE THREAD

**Issue by ericaryan** [TechAndCheck/zenodotus#643]
The "Download media metadata" link here doesn't lead to anything:
<img width="839" alt="Screenshot 2024-03-20 at 1 54 52 PM" src="https://github.com/TechAndCheck/zenodotus/assets/18681520/fab01aef-b8b7-4f91-928e-00d14cf10a4b">

Same with the JSON download button on author pages:
<img width="979" alt="Screenshot 2024-03-20 at 1 55 00 PM" src="https://github.com/TechAndCheck/zenodotus/assets/18681520/fe91924d-050f-48d3-b92c-a1952b2443e1">

**Comment by ericaryan** [comment]
We still need to remove the JSON links on the author pages, but we can do that next week

**Comment by cguess** [comment]
JSON links now disappear if there's no public posts for the author. If it's a mix, then it does show

**Comment by ericaryan** [comment]
This checks out! But when I click the JSON button, nothing downloads. Is that a staging thing?

<img width="312" alt="Screenshot 2024-04-01 at 5 03 10 PM" src="https://github.com/TechAndCheck/zenodotus/assets/18681520/62709b61-8c61-49e9-a4a0-b8d6157bb802">

**Comment by cguess** [comment]
The JSON button shouldn't exist if there's only private items for that poster. I tested and it works fine on regular posts. Can you give me a bit more detail?

**Comment by ericaryan** [comment]
Sure. I tested by looking at authors from my private MyVault posts, and there was no JSON button as expected.

But I also wanted to test the "mix" of public and private posts for author pages. So I picked an author that I knew already had posts in MediaVault (2 items archived). Then I went to their Instagram account and archived a new post to MyVault. The author page shows up as expected, with 3 items archived now, along with the JSON download button.

https://vault.staging.factcheckinsights.org/authors/instagram/b258cb7e-7ea7-4d08-b7f6-a5d3a8869974

However, when I actually click the JSON button to download, I get the "Site wasn't available" error you see in my screenshot above.

I just tested the JSON button for this author on prod and that doesn't work either, so I think we might have a problem that happens with this particular author?

https://vault.factcheckinsights.org/authors/instagram/b258cb7e-7ea7-4d08-b7f6-a5d3a8869974
